androidapp/DansFrappesRaps/.idea/deploymentTargetDropDown.xml

```diff
@@ -7,11 +7,11 @@
         <deviceKey>
           <Key>
             <type value="VIRTUAL_DEVICE_PATH" />
-            <value value="C:\Users\andre\.android\avd\Pixel_4a_API_33.avd" />
+            <value value="C:\Users\andre\.android\avd\Resizable_Experimental_API_33_1.avd" />
           </Key>
```

Comment:
Don't think we need this file, and it should be autogenerated by your android studio. Wanna just go ahead and delete it?